README.md

```diff
@@ -59,7 +59,7 @@ Configure AWS access:
 
 For example, to configure AWS access:
 
-Create config/initializers/aws.rb as follows:
+Create `config/initializers/aws.rb` as follows:
```

Comment:
Ideally this would be `config/initializers/dynamoid.rb` so the name matches the library it is for.  So many gems are AWS related, and AWS provides so many services, that aws.rb isn't very descriptive.